Adding kafka-replication script (#3256)
dimagi/commcare-cloud: docs/services/kafka.md

    ```

See https://kafka.apache.org/documentation.html#basic_ops_cluster_expansion for more details.

### Replication

> for setting up the replication on exxiting topic we make use of a helper script for easier use.

Reviewer: ```suggestion
For setting up the replication on existing topic we make use of a helper script which has the following capabilities:
```

> What it can do:

Reviewer: ```suggestion
```

* increase replication for existing topics
* decrease replication factor for existing topics
* remove all replicas from a particular broker so it can be decomissioned
* balance leaders

> for details on how to use this tool please see [kafka-reassign-tool](https://github.com/dimas/kafka-reassign-tool)

Reviewer: ```suggestion
For details on how to use this tool please see [kafka-reassign-tool](https://github.com/dimas/kafka-reassign-tool)
```

    ```

See https://kafka.apache.org/documentation.html#basic_ops_cluster_expansion for more details.

### Replication

#### [kafka-reassign-tool](https://github.com/dimas/kafka-reassign-tool)

> A helper script for Kafka to make it easier changing replicas for existing topics

Reviewer: could give a very short overview here and then point to the github docs for more detail rather than copying the docs here.

Author: Updated
